main.py

```diff
@@ -0,0 +1,220 @@
+    print(f'Humidity: {high_humidity["temperature"] or 0}% on {datetime.strptime(high_humidity["date"], "%Y-%m-%d").strftime("%B, %Y")}')
+
+def displayMonthlyDataCalculation(year, month, bar_count):
+    """
+    display records of single month
+    """
+    files_data = ReadFilesData(year, month)
+    temperature_calculation_obj = TemperatureCalculation(files_data)
+    
+    avg_high_temp = temperature_calculation_obj.requiredTemperatureCalculation("avg_H")
+    print(f'Highest Average: {avg_high_temp["temperature"] or 0}C')
+    
+    avg_low_temp = temperature_calculation_obj.requiredTemperatureCalculation("avg_L")
+    print(f'Lowest Average: {avg_low_temp["temperature"] or 0}C')
+        
+    avg_mean_humidity = temperature_calculation_obj.requiredTemperatureCalculation("avg_Hu")
+    print(f'Average Mean Humidity: {avg_mean_humidity["temperature"] or 0}%')
+
+    displayMonthlyReportBars(temperature_calculation_obj, bar_count = bar_count)
+    
+def displayMonthlyReportBars(temperature_calculation_obj, bar_count = 2):
+    """
+    display report of month or year in 1 or 2 bars which given by the user
+    """
+    print(f'\n{months[temperature_calculation_obj.data.month]} {temperature_calculation_obj.data.year}')
+    swap = True
+    display_record = {}
+    for value in temperature_calculation_obj.singleDateRecordGenerater():
+        splitted_date  = value and value["date"].split('-')
+        if(not splitted_date): 
+            swap = not swap
+            continue
+            
+        display_record["date"] = splitted_date[2]
+        temperature = value and value["value"]
+        display_record[(swap and 'max_temp') or 'min_temp'] = temperature
+        signs = ''
+        
+        for value in range(int(display_record[(swap and 'max_temp') or 'min_temp'])):
+            if(swap):
+                signs += f'{RED}+'
+            else:
+                signs += f'{BLUE}+'
+                
+        display_record[(swap and 'max_temp_sign') or 'min_temp_sign'] = signs
+        if bar_count < 2:
+            if(not swap): print(f'{display_record["date"]} {display_record["min_temp_sign"]}{display_record["max_temp_sign"]}{RESET} {display_record["min_temp"]}-{display_record["max_temp"]}')
+        else: 
+            if(swap):
+                print(f'{display_record["date"]} {display_record["max_temp_sign"]}{RESET} {display_record["max_temp"]}')
+            else:
+                print(f'{display_record["date"]} {display_record["min_temp_sign"]}{RESET} {display_record["min_temp"]}')
+                
+        swap = not swap
+    
+def main():
+    
+    for file_date in sys.argv[1:]:
+        splitted_file_date = file_date.split('/')
+        try:
+            if len(splitted_file_date) >= 2:
+                bar_count = int(input("Please Enter Number of Report bar_counts 1-2: "))
+                displayMonthlyDataCalculation(splitted_file_date[0], splitted_file_date[1], bar_count)
+                print('')
```

Comment:
Is that Command Line Argument update in progress?

Reply:
command arguments bug is fixed